src/diem/offchain/types/command_types.py

```diff
@@ -124,6 +137,19 @@ class OffChainErrorObject:
     message: typing.Optional[str] = datafield(default=None)
 
 
+@dataclass(frozen=True)
+class OffChainResultObject:
```

Comment:
change this to ReferenceIDCommandResultObject, may also update https://github.com/diem/dip/pull/164/files "CommandResultObject" to use same name, 

When there are multiple result objects, it is better we don't mix their fields.